README.md

```diff
@@ -1,4 +1,4 @@
-# mathlib4
+# mathlib4 aaa
```

Comment:
testing that this still works from review comments from branch PRs! Do not merge.
maintainer merge?